examples/errorprone/StringFmtInPrintMethodsCheck.java

```diff
@@ -60,10 +60,11 @@
         "github.com/google/startup-os/tree/master/examples/errorprone#StringFmtInPrintMethodsCheck")
 public class StringFmtInPrintMethodsCheck extends BugChecker
     implements MethodInvocationTreeMatcher {
-
+  @SuppressWarnings({"checkstyle:AbbreviationAsWordInName", "checkstyle:MemberName"})
   private Matcher<ExpressionTree> PRINT_METHOD =
```

Comment:
Could these also be static?

Reply:
@oferb yep, fixed in https://github.com/google/startup-os/pull/490/commits/630402f6a6a747564fc9d1b341bb508efae0591a